drivers/crypto/se05x.c

```diff
@@ -0,0 +1,266 @@
+/****************************************************************************
+ * drivers/crypto/se05x.c
```

Comment:
why not move into pnt folder

Reply:
The pnt folder only contains the external 'plug&trust' library and the needed wrappers. The intention was to seperate that from the ioctl functions